lesson26/src/js/login.js

```diff
@@ -0,0 +1,69 @@
+const userIdOfInput = document.querySelector(".js-form-userid");
+const passwordOfInput = document.querySelector(".js-form-password");
+const formElements = [userIdOfInput,passwordOfInput];
+const submitButton = document.getElementById("js-submit-button");
+
+const isNotEmptyOfInput = target => target.value.trim() !== "";
+const isNotEmptyOfAllInput = () => {
+    let result = false;
+    formElements.forEach(element => {
+        element.value.trim() !== "" ? result = true : result = false;
+    })
```

Comment:
IMO
You can also write like this
```suggestion
    formElements.forEach(element => {
       result = isNotEmptyOfInput(element)
    })
```